### DISCUSSION TITLE
GRPC reflection api

### DISCUSSION THREAD

**Discussion by haritbe** [grpc/grpc-java#11101] [General]
Hi Team,

We are building a passthrough proxy server to acceept json payloads and invoke grpc services using GRPC reflection api to make a RPC to a backend grpc api internally.

While using the api we noticed the package name for reflection api has "v1alpha". Does this mean that this api is not yet generally available?

**Comment by ejona86** [comment]
The API is definitely being used in production, but yes, we need to migrate to the v1 API. We are looking at doing that maybe in the next few months. Most of the other languages support v1 already. There were no changes between v1alpha and v1, other than the name.

To support v1, we need to allow you to use both v1alpha and v1 simultaneously. So there will be API additions to allow using the new version.

**Reply by ejona86** [reply]
You can follow #6724 for v1 addition.
